KSPCommunityFixes/KSPCommunityFixes.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't know why this line is marked as changed.

Reply:
It also happens to me pretty frequently, IDEs fighting over file encoding I guess ? doesn't really matter...